src/features/chat/ui/ChatTriggerButton.tsx

```
@@ -5,5 +5,5 @@ import { useRouter } from 'next/navigation';
 
 export default function ChatTriggerButton() {
   const router = useRouter();
-  return <button onClick={() => router.push(PATHS.CHAT)}>이거 채팅버튼임</button>;
+  return <button onClick={() => router.push(PATHS.CHAT.BASE)}>이거 채팅버튼임</button>;
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)